python/pyspark/mllib/stat/_statistics.py

```diff
@@ -170,10 +190,29 @@ def corr(x, y=None, method=None):
         if not y:
             return callMLlibFunc("corr", x.map(_convert_to_vector), method).toArray()
         else:
-            return callMLlibFunc("corr", x.map(float), y.map(float), method)
+            return callMLlibFunc(
+                "corr", x.map(float), y.map(float), method  # type: ignore[arg-type]
```

Comment:
```RDD.map``` expect a ```Callable``` argument

Reply:
`float` is `Callable`. What is the exact error you get here if it is not ignored.

Reply:
`float` is `Callable`. What is the exact error you get here if it is not ignored?

Reply:
```python
Argument 1 to "map" of "RDD" has incompatible type "Type[float]"; expected "Callable[[Vector], Any]"  [arg-type]
```
I got that error if it is not ignored, thanks!

Reply:
In such case it would be better to `cast` `x`, as we expect it is going to be `RDD[float]`.

Reply:
thank you! updated